src/test/anovos/data_transformer/test_transformers.py

```diff
@@ -0,0 +1,183 @@
+import os
+import pytest
+from pytest import approx
+from pyspark.sql import functions as F
+from anovos.data_ingest.data_ingest import read_dataset
+from anovos.data_transformer.transformers import attribute_binning, monotonic_binning, cat_to_num_unsupervised, cat_to_num_supervised, z_standardization, IQR_standardization, normalization, imputation_MMM, imputation_sklearn, imputation_matrixFactorization, auto_imputation, autoencoder_latentFeatures, PCA_latentFeatures, feature_transformation, boxcox_transformation, outlier_categories
+sample_parquet = "./data/test_dataset/part-00001-3eb0f7bb-05c2-46ec-8913-23ba231d2734-c000.snappy.parquet"
+
+#scaling
+def test_z_standardization(spark_session):
+    df = read_dataset(spark_session, sample_parquet, "parquet")
+    odf = z_standardization(spark_session, df)
```

Comment:
For all the functions can few more assert statements be added ? 
testing different arguments and also some negative test cases for example to name a few for z_standardization

with pre_existing_models
odf = z_standardization(spark, df, list_of_cols=['active_days', 'signal_count', 'wifi_engagement'], pre_existing_model=True, model_path=result_path, print_impact=True),

#different output modes
odf = z_standardization(spark, df, list_of_cols=['active_days', 'signal_count', 'wifi_engagement'], 
                        output_mode='append', print_impact=True)
                       
# empty list_of_cols
odf = z_standardization(spark, df, list_of_cols=[])